doc/source/changelog.rst

```diff
@@ -1,8 +1,19 @@
+:orphan:
+
 .. _ref_release_notes:
 
 Release notes
 #############
 
+
+.. button-ref:: release-note
+    :ref-type: doc
+    :color: primary
+    :shadow:
+    :expand:
+
+    Go to latest release notes
```

Comment:
I am not a big fan of duplicating files. I understand we have changed the format. However, why having a `CHANGELOG` and a `RELEASE NOTES` page? For me, it is confusing.

Reply:
We should only have one page IMO...

Reply:
> I am not a big fan of duplicating files. I understand we have changed the format. However, why having a `CHANGELOG` and a `RELEASE NOTES` page? For me, it is confusing.
I will update the template and revert the files. Currently, there is a title-level inconsistency with the existing template. I intended to have 'What's New' as a section, but doing so creates a title-level inconsistency since the previous template treats its children as sub-subsections. That is the reason i kept the new template in new file.